src/tools/verificarToken.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const Token = require('./token')
+
+const verificarToken = (req, res, next) => {
+    const token = req.get("x-token") || ""
+    Token.compararToken(token).then(decoded => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
    Token.compararToken(token).then((decoded) => {
```